src/futhark-test.hs

```diff
@@ -260,6 +260,13 @@ compareResult _ (RunTimeFailure f) (SuccessResult _) =
 --- Test manager
 ---
 
+data TestStatus = TestStatus { _testStatusRemain :: [TestCase]
+                             , _testStatusRun :: [TestCase]
+                             , _testStatusFail :: Int
+                             , _testStatusPass :: Int
+                             , _testStatusCases :: Int
+                             }
+
```

Comment:
Don't underscore-prefix field names that are actually used.  It's just a hack to shut up GHC from warning about unused names, in those cases where the field names are only used to improve the readability of the record definition.